content/legal/privacy-notice.md

```diff
@@ -250,7 +248,7 @@ For full details on how Plausible processes data please refer to their [Privacy
 
 We may share your information when working with our suppliers in order to provide the Service.
 
-In addition, the Matrix.org homeserver is a decentralised and open service. This means that, to support communication between users on different homeservers or different messaging platforms, your username, display name and messages and files are sometimes shared with other services that are connected with the Matrix.org homeserver.
+In addition, the [matrix.org](https://matrix.org) homeserver is a decentralised and open service. This means that, to support communication between users on different homeservers or different messaging platforms, your username, display name and messages and files are sometimes shared with other services that are connected with the [matrix.org](https://matrix.org) homeserver.
```

Comment:
I suggest that the whole "matrix.org homeserver" term is linked to the new section on the website at https://matrix.org/homeserver/ something like this:

```suggestion
In addition, the [matrix.org homeserver](@/homeserver/) is a decentralised and open service. This means that, to support communication between users on different homeservers or different messaging platforms, your username, display name and messages and files are sometimes shared with other services that are connected with the [matrix.org homeserver](@/homeserver/).
```